### ISSUE TITLE
"Skip remaining rules" don't work in "Business rules for tickets"

### ISSUE THREAD

**Issue by Viktor74Yl** [glpi-project/glpi#13824]
### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Version

10.0.5

### Bug description

user1 on **location1** create ticket with **category1**

rule1 - 
Category - Name is **category1**
Technician Assign Support1
Skip remaining rules Assign Yes

rule2 - 
Requester location is **location1**
Technician Assign Support2

Test rules engine - 
Support1 Yes
Support2 Yes

But the second rule should not apply, since the first one applies and is told to skip remaining rules

**Comment by cconard96** [comment]
Skip remaining rules action seems to be working properly. The only issue I noticed was that this functionality wasn't reflected when testing the rules.

**Comment by cedric-anne** [comment]
Rule test functionnality fixed by #13850.

If problem is still present at another place, feel free to reopen and give more details about way to reproduce.

**Comment by Viktor74Yl** [comment]
10.0.6
Rule testing is now working correctly, thanks for the fix.

There was an error in the working rules, the "Skip remaining rules" does not work if Technician (or Technician group) Assign in Action
Ticket to TEST-CATEGORY, requester in TEST-LOCATION
```
TEST-RULE-1
Category - Complete name is TEST-CATEGORY
Technician Assign Support1
Skip remaining rules Assign Yes

TEST-RULE-2	
Requester location is TEST-LOCATION
Technician Assign Support2

result of created ticket - 
Assigned to
Support1
Support2

When **testing** the same rules, only Support1 is assigned
```

Thus if for example Urgency Assign - that the "Skip remaining rules" of other rules works.
```
TEST-RULE-1
Category - Complete name is TEST-CATEGORY
Urgency Assign Very high
Skip remaining rules Assign Yes
	
TEST-RULE-2
Requester location is TEST-LOCATION
Technician Assign Support2

result of created ticket - 
Urgency Very high
```
